components/SSI/Stake/AddFunds/index.tsx

```diff
@@ -133,45 +135,80 @@ function StakeAddFunds() {
         }
     }
 
-    const handleConnect = React.useCallback(async () => {
+    const fetchInfoBalance = async (id: string, addr?: string) => {
```

Comment:
this is also repeated code, for wallets info? @devwistkey - let's have it in its own component

Reply:
Yes, done